docs/UserGuide.md

```diff
@@ -94,6 +94,9 @@ that you key in. Any success, error or warning messages will be displayed in thi
 
 Every item is made up of an item name, location and quantity. Expiry date and tags are optional. 
 
+Expiry date of an item will be shown in orange if the item is expiring in 3 days. Expiry date of an item will be shown
+in red if the item has expired.
+
 2 items are identical when they have the exact same name, location and expiry date.
```

Comment:
I think we should also add in a section to explain the difference between similar and identical items. Maybe, we can add a section for that or define them in the glossary.

Reply:
we do have that in the UG alr, it was updated by @kumsssss in the previous UG PR :)